.husky/pre-push

```diff
@@ -1,3 +1,7 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+
 #!/bin/sh
```

Comment:
This only means anything if it's the first line of the file (in which case it hints what shell to use if executing the file without being explicit).

Reply:
Whoops, blindly copy pasted haha